cdap-ui/app/cdap/components/EntityListView/EntityListInfo/index.js

```diff
@@ -25,8 +29,18 @@ export default class EntityListInfo extends Component {
     super(props);
   }
   handlePageChange(data) {
-    let clickedIndex = data.selected+1;
-    this.props.onPageChange(clickedIndex);
+    let currentPage = data.selected + 1;
+    let pageSize = SearchStore.getState().search.limit;
+    let offset = data.selected * pageSize;
+    SearchStore.dispatch({
+      type: SearchStoreActions.SETCURRENTPAGE,
+      payload: {
+        currentPage,
+        offset
+      }
+    });
+    search();
+    updateQueryString();
```

Comment:
should's this action dispatch automatically trigger these other actions (search and updateQueryString)?